src/app/(with-nav-footer)/(mypage)/my-activities/add-activities/_components/AddActivities.tsx

```diff
@@ -0,0 +1,176 @@
+    control,
+    handleSubmit,
+    setValue,
+    reset,
+    formState: { errors, isValid },
+  } = methods;
+
+  const [bannerImageUrl, setBannerImageUrl] = useState<string>('');
+  const [subImageUrls, setSubImageUrls] = useState<string[]>([]);
+
+  const onSubmit = (data: CreateActivityParams) => {
+    const finalData = {
+      ...data,
+      bannerImageUrl,
+      subImageUrls,
+    };
+
+    createActivity(finalData);
+    toast.success('내 체험 등록 성공');
+    reset();
+  };
+
+  return (
+    <FormProvider {...methods}>
+      <form
+        className='space-y-6'
+        onSubmit={handleSubmit(onSubmit, (invalidData) => {
+          console.log(invalidData);
+        })}
+      >
+        <div className='flex items-center justify-between'>
+          <h2 className='text-2xl font-bold'>내 체험 등록</h2>
+          <Button type='submit' disabled={isPending || !isValid} className='px-[20px] py-[11px]'>
+            {isPending ? '등록 중...' : '등록하기'}
+          </Button>
+        </div>
+
+        <Input
+          label='제목'
+          placeholder='제목을 입력하세요'
+          {...register('title')}
+          error={errors.title?.message as string}
+        />
+
+        <label className='text-black-100 inline-flex items-center gap-1 text-lg font-medium'>카테고리</label>
+        <Controller
+          control={control}
+          name='category'
+          render={({ field }) => (
+            <FilterDropdown
+              label='카테고리'
+              options={categoryOptions}
+              onSelect={(option) => field.onChange(option?.label || '')}
+              icon={arrowFilterDropdown2}
+              buttonClassName='border lg:w-[792px] w-[343px] md:w-[429px] text-black-100 border-gray-800 rounded-lg md:justify-between px-[15px] py-[15px]'
+              dropdownClassName='rounded-xl lg:w-[792px] w-[343px] md:w-[429px] border border-gray-300 bg-white drop-shadow-sm'
+              optionClassName='text-md md:text-lg h-[41px] lg:w-[792px] w-[343px] md:w-[429px] leading-[41px] md:h-[58px] md:leading-[58px]'
+              includeAllOption={false}
+              iconVisibleOnMobile={false}
+              value={field.value}
+            />
+          )}
+        />
+        {errors.category && <p className='text-sm text-red-500'>{errors.category.message}</p>}
+
+        <Input
+          label='설명'
+          placeholder='설명을 입력해주세요'
+          className='h-[260px]'
+          {...register('description')}
+          error={errors.description?.message as string}
+        />
+
+        <Controller
+          control={control}
+          name='address'
+          render={({ field }) => (
+            <AddressFind value={field.value} onChange={field.onChange} error={errors.address?.message as string} />
+          )}
+        />
+
+        <Input
+          label='가격'
+          placeholder='숫자만 입력'
+          type='number'
+          {...register('price', { valueAsNumber: true })}
+          error={errors.price?.message as string}
+        />
```

Comment:
```suggestion
        <Input
          label='가격'
          placeholder='숫자만 입력'
          type='number'
          {...register('price', { valueAsNumber: true })}
          error={errors.price?.message as string}
          className='appearance-none [&::-webkit-inner-spin-button]:appearance-none [&::-webkit-outer-spin-button]:appearance-none'
          onWheel={(e) => e.currentTarget.blur()}
        />
```
가격 인풋에 스크롤이나 증감버튼이 필요하지 않을 것 같은데 이렇게 수정하면 버튼과 휠이 막힙니다!
그리고 숫자를 기입 후 전부 지우면 'Expected number, received nan' 라고 에러가 뜨는데 한글로 바꾸는게 좋을 것 같아용